services/brig/test/integration/Federation/End2end.hs

```diff
@@ -238,6 +239,29 @@ testAddRemoteUsersToLocalConv brig1 galley1 brig2 = do
     )
     !!! (const 200 === statusCode)
 
+  -- test GET /conversations/:backend1Domain/:cnv
+  testQualifiedGetConversation galley1 "galley1" alice bob convId
+  testQualifiedGetConversation galley2 "galley2" alice bob convId
+  where
+    testQualifiedGetConversation galley backend alice bob convId = do
+      let backend1Domain = qDomain (userQualifiedId alice)
+          -- FUTUREWORK add qualified conversation Id to Conversation data type, then use that from the conversation creation response
+          qualifiedConvId = Qualified convId backend1Domain
+      res' <- getConvQualified galley (userId bob) qualifiedConvId <!! (const 200 === statusCode)
+      let conv' = responseJsonUnsafeWithMsg (backend <> "- get /conversations/domain/cnvId") res'
+          actual' = cmOthers $ cnvMembers conv'
+          expected' = [OtherMember (userQualifiedId bob) Nothing roleNameWireAdmin]
+      liftIO $ actual' @?= expected'
+
+    getConvQualified :: Galley -> UserId -> Qualified ConvId -> Http ResponseLBS
```

Comment:
This is a generally useful utility function so how about making it a top-level exported function in a `Util` module?

Reply:
I based my PR on this PR, which is where I already made the change I proposed here. See https://github.com/wireapp/wire-server/pull/1594/files.

Reply:
thanks, I backported your change.